stringOperations.java

```diff
@@ -0,0 +1,122 @@
+import java.util.Scanner;
+
+public class stringOperations {
+    static Scanner input = new Scanner(System.in);
+    static String originalString;
+
+    public static void main(String[] args) {
+
+        System.out.println("Enter a String: ");
+        originalString = input.nextLine();
+        if (originalString == null || originalString.isEmpty()) {
+            System.out.println("Error: String can't be empty");
+            return;
+        }
+        
+        switchCase();
+        // String encryptedMessage = shiftVariability(originalString);
+        // System.out.println("Encrypted Message: " + encryptedMessage);
+    }
+
+    public static String shiftVariability(String mainString) {
```

Comment:
describe each method in brief through comments including short definition , input parameters and return values .